custom_components/tesla_custom/climate.py

```diff
@@ -80,15 +80,13 @@ def target_temperature(self):
         """Return the temperature we try to reach."""
         return self.tesla_device.get_goal_temp()
 
-    @TeslaDevice.Decorators.check_for_reauth
```

Comment:
We may have to double check in teslajsonpy but I think this should still be good and will help catch a reauth need.